docs/instrumentation_examples.md

```diff
@@ -0,0 +1,214 @@
+# Armory Instrumentation Examples: Measuring Experiment Artifacts Using Probes and Meters
```

Comment:
I think we should also update this link to `metrics.md#instrumentation`

Reply:
resolved with c24837a7